Dockerfile

```diff
@@ -14,10 +11,14 @@ RUN luet install -y \
     system/cos-setup \
     system/immutable-rootfs \
     system/grub2-config \
+    system/grub2-efi-image \
+    system/grub2-artifacts \
     selinux/k3s \
     selinux/rancher \
     utils/nerdctl \
-    toolchain/yq
+    toolchain/yq \
+    utils/k9s \
+    toolchain/elemental-cli
```

Comment:
We should install packages from the elemental OBS repo for maximum support. Is it possible to switch?